homeassistant/components/bmw_connected_drive/__init__.py

```diff
@@ -117,7 +118,7 @@ async def _async_update_all(service_call=None):
 
     def _update_all() -> None:
         """Update all BMW accounts."""
-        for entry in hass.data[DOMAIN].values():
+        for entry in [e for k, e in hass.data[DOMAIN].items() if k != DATA_HASS_CONFIG]:
```

Comment:
Please use a separate dict nested under `hass.data[DOMAIN]` for the entry data, accounts and listeners etc.

Reply:
I hope `DATA_ENTRIES = "entries"` is the correct wording, otherwise this could be changed quickly.